website/content/en/docs/contribution-guidelines/release.md

```diff
@@ -235,25 +242,51 @@ Commit the following changes:
 - `website/content/en/docs/migration/v1.3.0.md`: commit changes (created by changelog generation).
 - `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
 
-_(Non-patch releases only)_ Lock down the master branch to prevent further commits between this and step 4. See [this section](#locking-down-branches) for steps to do so.
+#### Non-patch releases
+
+Lock down the master branch to prevent further commits between this and step 4. See [this section](#locking-down-branches) for steps to do so.
+
+Commit the following changes:
+
+- `website/config.toml`: update `version_menu = "Releases"` with the patch-less version string `version_menu = "v1.3"`,
+and add the following lines under `[[params.versions]]` for `master`:
+  ```toml
+  [[params.versions]]
+    version = "v1.3"
+    url = "https://v1-3-x.sdk.operatorframework.io"
```

Comment:
As mentioned [above](https://github.com/operator-framework/operator-sdk/pull/3383/files#diff-c7aebce28b63411bb609012adc8ab83eR186-R187), this link won't work (and will fail CI) unless Netlify is already building this branch. The configuration on the netlify side cannot be done completely ahead of time though, the github branch must be created and pushed first. 

I think if we add a `git push` [here](https://github.com/operator-framework/operator-sdk/pull/3383/files#diff-c7aebce28b63411bb609012adc8ab83eR227) that should cover it.

Reply:
Done